pandas/core/groupby/ops.py

```diff
@@ -193,15 +193,20 @@ def apply(self, f, data, axis=0):
         group_keys = self._get_group_keys()
         result_values = None
 
-        # oh boy
-        f_name = com.get_callable_name(f)
-        if (
-            f_name not in base.plotting_methods
+        sdata = splitter._get_sorted_data()
+        if sdata.ndim == 2 and np.any(sdata.dtypes.apply(is_extension_array_dtype)):
```

Comment:
I would move this to the Splittler class something like

```
@cache_only
def has_extension_array(self):
     return self.data.dtypes.apply(...)
```

and can handle this on Series & Frame splitter

Reply:
yah, this function is definitely getting refactored in a later pass.  id like to get the exceptions sorted out before trying to decide about the refactor